rDNA/R/rDNA.R

```diff
@@ -1806,4 +1807,289 @@ print.dna_barplot <- function(x, trim = 30, attr = TRUE, ...) {
   x2$Entity <- sapply(x2$Entity, function(e) if (nchar(e) > trim) paste0(substr(e, 1, trim - 1), "*") else e)
   class(x2) <- "data.frame"
   print(x2)
+}
+
+#' Plot dna_barplot object.
+#'
+#' Plot barplots generated from dna_barplot.
```

Comment:
Put the function name in a code and link markup like elsewhere in the code? Same thing further below.